src/aui-form-builder/tests/unit/js/aui-form-builder-field-base-tests.js

```diff
@@ -319,11 +319,15 @@ YUI.add('aui-form-builder-field-base-tests', function(Y) {
 
             Y.Assert.isTrue(Y.one('.form-builder-field-settings-panel-advanced-content').hasClass('toggler-content-expanded'));
 
-            Y.one('.form-builder-field-settings-panel-advanced-button').simulate('click');
-            Y.Assert.isTrue(Y.one('.form-builder-field-settings-panel-advanced-content').hasClass('toggler-content-collapsed'));
+            Y.one('.form-builder-field-settings-panel-toggler-advanced').simulateGesture('tap');
```

Comment:
Actually I hadn't needed to change the event click by tap. It was necessary only make a little change in the code. But I will write a tests using both events.